Admin/Pool.php

```diff
@@ -20,22 +20,49 @@
  */
 class Pool
 {
+    /**
+     * @var ContainerInterface|null
+     */
     protected $container = null;
 
+    /**
+     * @var array
+     */
     protected $adminServiceIds = array();
 
+    /**
+     * @var array
+     */
     protected $adminGroups = array();
 
+    /**
+     * @var array
+     */
     protected $adminClasses = array();
 
-    protected $templates    = array();
+    /**
+     * @var array
+     */
+    protected $templates = array();
 
-    protected $assets       = array();
+    /**
+     * @var array
```

Comment:
It was used: https://github.com/sonata-project/SonataAdminBundle/commit/45b2ad31b50bacecc8eabf9a294e9e80889a7306 but some code has been removed.